doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1111,6 +1112,9 @@ Applying theorems
 
       apply Rmp.
 
+.. todo PR the following title isn't the greatest.  Perhaps more like "trivial tactics"
+   or "simple tactics"???
```

Comment:
Simple tactics or Basic tactics would probably be appropriate titles for the entire chapter actually. Let's discuss this separately from this PR.